bundles/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/internal/handler/TeslaAccountHandler.java

```diff
@@ -269,33 +267,11 @@ private ThingStatusInfo authenticate() {
         }
 
         if (hasExpired) {
-            String username = (String) getConfig().get(CONFIG_USERNAME);
```

Comment:
Should/could we delete CONfIG_USERNAME? And password?

Reply:
Certainly so! Done.